resources/entrypoints/main.sh

```diff
@@ -82,11 +82,23 @@ else
 			# Make sure the process does not exit due to some
 			# failure.
 			true
+		}
+		while true; do
+			ensure-group-permissions
 			sleep 5
 		done &
```

Comment:
When running this, my terminal looks as follows:

```
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/1/T15F1.ctl' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/1/T15F0.ctl' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/1/T17F2.ctl' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/10/T17F1.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/10/T17F0.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/10/T17F3.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/10/T17F2.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/8/T17F1.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/8/T17F0.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/8/T17F3.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/8/T17F2.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/15/T17F1.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/15/T17F0.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/15/T17F3.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/15/T17F2.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/12/T17F1.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/12/T17F0.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/12/T17F3.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/12/T17F2.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/13/T17F1.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/13/T17F0.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/13/T17F3.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/13/T17F2.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/14/T17F1.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/14/T17F0.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/14/T17F3.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
ndbd_1    | mode of '/srv/hops/mysql-cluster/ndb_data/1/ndb_1_fs/LCP/14/T17F2.Data' changed from 0644 (rw-r--r--) to 0664 (rw-rw-r--)
```

Basically I can barely read anything else anymore. Is there any way of hiding this?

Reply:
This happens a lot when benchmarks are run. So it swallows the api container logs. And it is even run when omitting `-lv` (I don't mind that, I just mind the printing).

Reply:
If possible maybe only print stderr

Reply:
I activated printing on purpose, but you're right that it's better not to print. I'll change it.

Reply:
Fixed.